packages/cdktf-cli/templates/go/go.mod

```diff
@@ -0,0 +1,7 @@
+module cdk.tf/go/stack
+
+go 1.16
```

Comment:
Is this a minimum or exact version?

Reply:
Good catch. According to the [docs](https://golang.org/ref/mod#go-mod-file-go), it is the minimum version.
I'm going to see if I can lower it.

Reply:
Going to leave it at `1.16` as JSII seems to require that version.

for reference:
https://github.com/aws/jsii/blob/e84e70f5549a6a0382591aa7a735086fa3e65154/packages/jsii-pacmak/lib/targets/go/package.ts#L28
https://github.com/aws/jsii/blob/23f3582fafaee2b7a64c92d6d426783b31f5aea4/superchain/README.md#included-language-sdks